### DISCUSSION TITLE
Wrong translation for Russian language for minutes

### DISCUSSION THREAD

**Discussion by natakazakova** [squidfunk/mkdocs-material#6906] [Q&A] [answered]
Hi! I've tried the Russian version of mkdocs-material and I've noticed the wrong translation for numbers.
![image](https://github.com/squidfunk/mkdocs-material/assets/17986517/dbfc87c8-1aca-4b99-a8e0-8bd5b5a028c1)
In case 2-4 that should be "минуты":
1 минута
2-4 минуты (22-24, 32-34 and so on will also be "минуты")
5-... минут (15 минут, 20 минут, 36 минут and so on).

I think it also can be in Serbian and other Slavic languages.

How can I update the translation for this? Can someone help?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @natakazakova,
Polish had the same issues, 1 minu**ta**, 2-4 minu**ty**, 5 minu**t**, but there are only 2 variants for the readtime display:
https://github.com/squidfunk/mkdocs-material/blob/2f1b2e950040a89e70a5f193de762388206fb8fa/src/templates/partials/languages/pl.html#L50-L51
So it was solved with a generic string and abbreviation of minute, Reading time: 1 min, Reading time: # min, which works in Polish.

As for how to update, if you only want to provide an update to the translations:
- Please use this [`Add translation`](https://github.com/squidfunk/mkdocs-material/issues/new?assignees=&labels=change+request&projects=&template=04-add-translations.yml&title=Update+Russian+translations) issue template, 
- Copy over the whole macro from the [`ru.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/templates/partials/languages/ru.html) file
- Make relevant changes to the translations
- Optional: Add a short comment in the Issue's title to make it unique
- Add a comment after publishing the Issue, why the change was made (you can reference this Discussion 👌)

Reference:
- https://github.com/squidfunk/mkdocs-material/issues/4962

If you plan to be extra thorough and commit the changes yourself with a PR, follow the theme development guide:
- https://squidfunk.github.io/mkdocs-material/customization/#theme-development

**Reply by squidfunk** [reply]
We'll likely move to proper translations via `*.po` files and `gettext` in the future, which will allow for more accuracy. Please understand, however, that it is currently not our highest priority, as we're currently working on bigger features.

**Reply by natakazakova** [reply]
Thanks! I've created the issue: [#6959](https://github.com/squidfunk/mkdocs-material/issues/6959)
